packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -178,6 +213,19 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
           const isEnded = log.get('ended')
 
           if (isEnded) {
+            if (log.get('state') === 'failed') {
+              const err = log.get('err')
+              let parsedError = correctStackForCrossDomainError(err, switchToDomainUserInvocationStack)
+
+              // The toJSON method on the Cypress.Log converts the 'error' property to 'err', so when the log gets
+              // serialized from the SD to the PD, we need to do the opposite
+              log.set('error', parsedError)
+              if ((logAttrs.consoleProps as any)?.Error) {
```

Comment:
I wonder if we can coerce this at all to call `wrapConsoleProps` by setting `log.set` here, or just calling `log.error` and then maybe calling `wrapConsoleProps` so we can avoid most of this code. Any thoughts on this?